lib/model/content.dart

```diff
@@ -74,6 +75,12 @@ mixin UnimplementedNode on ContentNode {
 
 sealed class ZulipMessageContent {}
 
+class PollContent implements ZulipMessageContent {
```

Comment:
While this is pretty short, I'm not sure if we need a better way to organize `ZulipMessageContent`, `PollContent`, and `MessageContent`. The file `lib/model/content.dart` is predominantly about our AST for Zulip HTML content.